libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/model/eh/RoadObjectPassInfo.kt

```diff
@@ -0,0 +1,49 @@
+package com.mapbox.navigation.core.trip.model.eh
+
+import com.mapbox.navigation.base.trip.model.roadobject.RoadObjectType
+
+/**
+ * RoadObjectPassInfo
```

Comment:
Could you elaborate what is a pass info?

Reply:
done